public/app/features/dashboard/components/PanelEditor/OptionsPaneCategory.tsx

```diff
@@ -101,12 +101,23 @@ export const OptionsPaneCategory: FC<OptionsPaneCategoryProps> = React.memo(
         ref={ref}
       >
         <div className={headerStyles} onClick={onToggle} aria-label={selectors.components.OptionsGroup.toggle(id)}>
-          <div className={cx(styles.toggle, 'editor-options-group-toggle')}>
+          <Button
+            size="sm"
```

Comment:
Storybook tells me a `Button` has a `xs` option for this prop that might make the rows closer to what we currently have but when I changed it , it defaulted to `md` : looks like `xs` is not a valid `size` 🤷‍♀️